### ISSUE TITLE
New merge references packages that aren't in apt.tcpcloud.eu

### ISSUE THREAD

**Issue by mediumstack** [salt-formulas/salt-formula-galera#3]
The latest commit 2a8cc4a makes changes to map.jinja, the following packages are different than before:

mysql-server-wsrep-5.6, galera-3 - these packages do not currently exist in apt.tcpcloud.eu, breaking deployment.

**Comment by pupapaik** [comment]
Yes, we decided to push a new tag version to 1.0 and use different packages, which are not compatible with old one and must be under testing. For current deployment please use older version.
